Adjust code and explanations to new testing guidelines (#1270)
cosmos/sdk-tutorials: hands-on-exercise/1-ignite-cli/4-create-message.md

## Unit tests

The code of this section was created by Ignite CLI, so there is no point in testing it. However, since you are going to adjust the keeper to do what you want, you should add a test file for that.

First, recall your address constants in the `keeper_test` package:

```go [https://github.com/cosmos/b9-checkers-academy-draft/blob/create-game-msg/x/checkers/keeper/common_test.go]
package keeper_test

import "github.com/b9lab/checkers/x/checkers/testutil"

const (
    alice = testutil.Alice
    bob   = testutil.Bob
    carol = testutil.Bob
)
```

> Next, add `keeper/msg_server_create_game_test.go`, declared with `package keeper_test`, and add in:

Reviewer: @xavierlepretre maybe: "Next, add into `keeper/msg_server_create_game_test.go`, declared with `package keeper_test`:" ?

Author: It is an entirely new file so it cannot be "add into".

```suggestion
Next, create a new `keeper/msg_server_create_game_test.go`, declared with `package keeper_test`:
```
